mkimage

```diff
@@ -221,7 +221,7 @@ done
 # Remove the aux-cache as it isn't reproducible. It doesn't seem to
 # cause any problems to remove it.
 rm "$rootfsDir/var/cache/ldconfig/aux-cache"
-find "$rootfsDir/usr/share/doc" -mindepth 1 -not -name copyright -not -type d -delete
+find "$rootfsDir/usr/share/doc" -mindepth 2 -not -name copyright -not -type d -delete
```

Comment:
> However, the structure of /usr/share/doc is such that the first
> level is a per-package directory (or symlink) and the directory
> contains the files. We therefore want to only consider files in the
> package directories, so look a level further down the tree before
> deleting.
This is probably worth mentioning in a comment, right?

Reply:
Yeah, good point.